wagtailio/project_styleguide/templates/patterns/includes/header.html

```diff
@@ -13,12 +13,24 @@
 
 
     <div class="header__col-two">
-        <button class="header__button header__button--search" aria-label="Open the search" data-site-search>
+        {# Desktop Search #}
+        <button class="header__button header__button--search u-desktop-only" aria-controls="site-search-modal" aria-label="Open the search" data-site-search-desktop>
             <svg class="header__search-icon" aria-hidden="true">
                 <use xlink:href="#magnifying-glass"></use>
             </svg>
         </button>
 
+        {# Mobile Search #}
+        <button class="header__button header__button--search u-mobile-only" aria-expanded="false" aria-controls="mobile-search-form" aria-label="Open the search" data-site-search-mobile>
+            <svg class="header__search-icon header__search-icon--open" aria-hidden="true">
+                <use xlink:href="#magnifying-glass"></use>
+            </svg>
+
+            <svg class="header__search-icon header__search-icon--close" aria-hidden="true">
```

Comment:
Note: I think aria-hidden might not be the correct usage here?

Reply:
Similar to the above - the aria-label on the parent button will be announced so there's no need to have this icon in the accessibility tree.